<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectChangedExternDialog</class>
 <widget class="QDialog" name="ProjectChangedExternDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dialogLabel">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonYes">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&amp;Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNo">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&amp;No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDetails">
       <property name="text">
        <string>&amp;Details...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>ProjectChangedExternDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNo</sender>
   <signal>clicked()</signal>
   <receiver>ProjectChangedExternDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonYes</sender>
   <signal>clicked()</signal>
   <receiver>ProjectChangedExternDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDetails</sender>
   <signal>toggled(bool)</signal>
   <receiver>ProjectChangedExternDialog</receiver>
   <slot>onShowDetails(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onShowDetails(bool)</slot>
 </slots>
</ui>
